src/console/types.ts

```diff
@@ -20,6 +20,7 @@ export interface MirrorConfig {
   enableRelay: boolean;
   autoSwitcherSettings?: AutoSwitcherSettings;
   useNicknameFolders: boolean;
+  consoleNick?: string;
```

Comment:
```suggestion
  nickname?: string;
```
Can we just call this `nickname`? I actually hate that somehow we ended up with `consoleNick` somewhere in our API but there's no reason why we need to use that name here.